lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -1032,7 +1033,16 @@ sub save_edit :Chained('base') :PathPart('save') :Args(0) {
             if ($field =~ /designer|producer/){
                 return $c->forward($c->view('JSON')) unless ($complat_user_admin || $value eq '');
             } elsif ($field eq "maintainer") {
-                return $c->forward($c->view('JSON')) unless $value = format_maintainer($c, $value, $ia, 0);
+                my $result = format_maintainer($c, $value, $ia, 0);
+               use Data::Dumper;
+               print Dumper $result;
```

Comment:
Maria, was just looking at this again.  Lines 1037-8 probably need to be removed.

On line 1039, what exactly are you trying to catch here?  A failed return from `format_maintainer`?  I think the logic here might be off:
``` Perl
$VAR1 = {
          'msg' => 'User has no GitHub account linked',
          'maintainer' => {}
        };
```
`maintainer` is always there, even if an empty hash, so it never gets in this `if`.  Is this why, when there is no github account, I get no error?  I see the message from `format_maintainer` come through but it never displays on the page afaict.

It seems to work fine if a user has a github login associated with their account.